src/format.ts

```diff
@@ -175,6 +175,7 @@ const rc = new RequestContext(
 )
 const request = rc.request.bind(rc)
 
+export { request }
```

Comment:
what do you think about making these exports consistent? either the existing style:

```ts
export const request = rc.request.bind(rc)
```

or making the existing one like this:

```ts
const setRequestHandler = rc.setRequestHandler.bind(rc)
const setResponseHandler = rc.setResponseHandler.bind(rc)
const request = rc.request.bind(rc)

export {request, setRequestHandler, setResponseHandler}
```
whichever one you prefer more?

Reply:
makes sense